erpnext/stock/get_item_details.py

```diff
@@ -281,7 +281,7 @@ def get_basic_details(args, item):
 		out.conversion_factor = 1.0
 	else:
 		out.conversion_factor = args.conversion_factor or \
-			get_conversion_factor(item.item_code, args.uom).get("conversion_factor") or 1.0
+			get_conversion_factor(item.item_code, args.uom or item.get('stock_uom')).get("conversion_factor", 1) or 1.0
```

Comment:
```suggestion
			get_conversion_factor(item.item_code, args.uom or item.get('stock_uom')).get("conversion_factor") or 1.0
```

seeing as how we already have `or 1.0`